media-sound/mixxx/mixxx-2.5.3.recipe

```diff
@@ -0,0 +1,179 @@
+SUMMARY="DJ Your Way, powerful Features For All DJs"
```

Comment:
Weird summary... :)
Maybe at least sentence-case?

Reply:
Grabbed from https://mixxx.org/ 
Probably could use better one, wasn't my primary focus. :)

Reply:
This sounds better? 

```
SUMMARY="Free and open source DJ software for Windows, macOS, Linux and Haiku"
```